src/main/scala/com/github/traviscrawford/spark/dynamodb/DynamoBackupJob.scala

```diff
@@ -15,6 +20,8 @@ object DynamoBackupJob extends Job {
 
   val output = flag[String]("output", "Path to write the DynamoDB table backup.")
 
+  val overwritePath = flag[Boolean]("overwritePath", false, "Set to true to overwrite output path.")
```

Comment:
Let's name this `overwrite`. Also, since we have a default value, we don't need to provide the flaggable type:

```
val overwrite = flag("overwrite", false, "Set to true to overwrite output path.")
```